pkg/accesscontrol/session.go

```diff
@@ -2,34 +2,37 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package dynamicaccess
+package accesscontrol
 
 import (
 	"crypto/ecdsa"
 	"errors"
 	"fmt"
 
 	"github.com/ethersphere/bee/v2/pkg/crypto"
-	"github.com/ethersphere/bee/v2/pkg/keystore"
+)
+
+var (
+	// ErrInvalidPublicKey is an error that is returned when a public key is nil.
+	ErrInvalidPublicKey = errors.New("invalid public key")
+	// ErrSecretKeyInfinity is an error that is returned when the shared secret is a point at infinity.
+	ErrSecretKeyInfinity = errors.New("shared secret is point at infinity")
 )
 
 // Session represents an interface for a Diffie-Helmann key derivation
```

Comment:
Another small thing I just noticed that it should be spelled: Diffie-Hellman

Reply:
fixed